src/website/app/demos/Pagination/examples/visibleRanges.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
```

Comment:
I am not really sure what this example is demonstrating.  Can it perhaps be turned into a unit test instead?  Alternatively, it needs a description and/or to show the code.

Reply:
Yes, probably better as a unit test; it was visually helpful while developing (for the edge-case of range of 1), but may be better off as a test from here on.